src/vmm/src/lib.rs

```diff
@@ -307,13 +310,17 @@ impl Vmm {
     pub fn resume_vm(&mut self) -> Result<()> {
         self.mmio_device_manager.kick_devices();
         self.broadcast_vcpu_event(VcpuEvent::Resume, VcpuResponse::Resumed)
-            .map_err(|_| Error::VcpuResume)
+            .map_err(|_| Error::VcpuResume)?;
+        self.instance_info.state = VmState::Running;
```

Comment:
this commit uses the `VmState`, which is introduced in the next commit, which makes the PR harder to review, and also uses both `state` types (enum and string)
I suggest we decouple the commits

Reply:
yes, this was a commit fixup miss. will fix it